library/macos/FLEViewController.m

```diff
@@ -33,6 +33,12 @@
 
 static const int kDefaultWindowFramebuffer = 0;
 
+// Android KeyEvent constants from https://developer.android.com/reference/android/view/KeyEvent
+static const int kAndroidMetaStateShift = 1;
+static const int kAndroidMetaStateAlt = 2;
+static const int kAndroidMetaStateCtrl = 4096;
+static const int kAndroidMetaStateMeta = 65536;
```

Comment:
I'd prefer to use the more common convention of expressing bit flags via shifting, rather than as decimal numbers, since it more clearly expresses how they are used. E.g., `...StateMeta = 1 << 16`.

Reply:
Done.